modules/collabora_online_group/tests/src/Kernel/AccessTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\collabora_online_group\Kernel;
+
+use Drupal\group\PermissionScopeInterface;
+use Drupal\Tests\collabora_online\Traits\MediaCreationTrait;
+use Drupal\Tests\collabora_online_group\Traits\GroupRelationTrait;
+use Drupal\Tests\group\Kernel\GroupKernelTestBase;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\Tests\user\Traits\UserCreationTrait;
+use Drupal\user\RoleInterface;
+
+/**
+ * Tests Collabora group access.
+ */
+class AccessTest extends GroupKernelTestBase {
+
+    use MediaTypeCreationTrait;
+    use UserCreationTrait;
+    use MediaCreationTrait;
+    use GroupRelationTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected static $modules = [
+        'file',
+        'media',
+        'image',
+        'group',
+        'groupmedia',
+        'collabora_online',
+        'collabora_online_group',
+        'user'
+    ];
+
+    /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->installEntitySchema('file');
+    $this->installEntitySchema('media');
+    $this->installEntitySchema('group_role');
+    $this->installSchema('file', ['file_usage']);
+  }
+
+    /**
+     * Tests that access to Collabora group permissions is handled.
+     *
+     * @param bool $expected_result
+     *   The access result.
+     * @param array $permissions
+     *   The global permissions granted to the user.
```

Comment:
For static analysis (IDE, PhpStan):
```suggestion
     * @param list<string> $permissions
     *   The global permissions granted to the user.
```

Reply:
Ok